R/resample.R

```diff
@@ -150,6 +150,12 @@ doResampleIteration = function(learner, task, rin, i, measures, weights, model,
     ms.test = vnapply(measures, function(pm) performance(task = task, model = m, pred = pred.test, measures = pm))
   }
   ex = extract(m)
+  if (show.info) {
+    mids = vcapply(measures, function(m) m$id)
+    for (i in 1:length(mids)) {
```

Comment:
Replace double loop (vcapply + for) with Map over measures and ms.test (or use vectorization of sprintf). Do not store return value. Use seq_along() instead of `1:length(x)`.